### DISCUSSION TITLE
How to run tests in order

### DISCUSSION THREAD

**Discussion by kidmortal** [vitest-dev/vitest#1933] [Q&A] [answered]
Im having a little trouble trying to use vitest along with flutter appium test.

![image](https://user-images.githubusercontent.com/18023467/187284160-8e74759d-fba4-4c8d-bc15-13bb9f0d7ce5.png)

![image](https://user-images.githubusercontent.com/18023467/187284179-8bc4e420-1fea-45ca-8155-02b889c7fed8.png)

the setup part should be the first to run, not the last, which is causing all other tests to fail

i need these tests to run in a specific order, any way to order it?

**Comment by sheremet-va** [comment, marked as answer]
Use `config.test.sequence.sequencer`

**Reply by kidmortal** [reply]
Thanks, that did the trick.
